src/app/layout.tsx

```diff
@@ -52,33 +51,45 @@ if ((!themeChosen && isDarkModePreferred) || themeChosen === "dark") {
           name="twitter:image"
           content="https://www.rustcrab.com/rust_lgo_720.png"
         />
-        <Script
-          async
-          src="https://www.googletagmanager.com/gtag/js?id=G-J6GRE0TKHY"
-        ></Script>
-        <Script id="google-analytics">
-          {`
-            window.dataLayer = window.dataLayer || [];
-            function gtag(){dataLayer.push(arguments);}
-            gtag('js', new Date());
-            gtag('config', 'G-J6GRE0TKHY');
-          `}
-        </Script>
-      </head>
-      <Head>
-        <title>Rustcrab - The Non-Crap Repo for Rust Developers</title>
-        <meta
-          name="description"
-          content="The best repository for Rust developers"
-        />
+
+        {process.env.NODE_ENV === "production" && (
```

Comment:
These scripts shouldn't be added to the application is we're in `dev` mode. They may create noise, or clutter *actual* app signals with dev-mode.